ProcessLib/BoundaryCondition/CreateBoundaryCondition.cpp

```diff
@@ -80,6 +81,16 @@ std::unique_ptr<BoundaryCondition> createBoundaryCondition(
             *config.component_id, integration_order, shapefunction_order,
             bulk_mesh);
     }
+
+    if (type == "NonuniformVariableDependantNeumann")
```

Comment:
dependent

Reply:
renaming is done in all files, variables, documentations, and other declarations (at least i hope so...)